.eslintrc.yml

```diff
@@ -1,10 +1,2 @@
 root: true
-extends: semistandard
-rules:
-  indent:
-    - error
-    - 4
-  camelcase: off
-  no-unused-vars:
-    - error
-    - args: after-used
```

Comment:
If repos choose to be stricter than our base, don't we want to maintain this for now? Maybe discuss bringing it into `@cordova/eslint-config` sometime?

Reply:
I went ahead and re-added it but also updated the settings to be more accurate to what we want with `standards` defaults.

This change should still be discussed and brought to the `cordova-eslint` so all repos are enforced.